features/step_definitions/help_relinking.rb

```diff
@@ -17,15 +17,21 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #++
 
-Given /^the help link setting points to the project (.+)$/ do |project_name|
-  project_name.gsub!("\"", '')
-  p = Project.find_by_name(project_name)
+Given(/^the help link setting points to help page$/) do
   new_values = Setting.plugin_openproject_help_link
-  new_values['help_link_target'] = "/projects/#{p.identifier}"
+  new_values["help_link_target"] = OpenProject::Info.help_url
   Setting.plugin_openproject_help_link = new_values
```

Comment:
that's a no-op.

`new_values["help_link_target"] = OpenProject::Info.help_url` expands to `new_values["help_link_target"] = Setting.plugin_openproject_help_link['help_link_target']` so this line becomes

```
Setting.plugin_openproject_help_link = Setting.plugin_openproject_help_link['help_link_target']`
```

 try to use more literal values when testing, like it was before. changing the the `default` help link target to `foo` will still pass your tests.